DependencyInjection/Configuration.php

```diff
@@ -118,6 +117,13 @@ public function getConfigTreeBuilder(): TreeBuilder
                         })
                     ->end()
                     ->children()
+                        ->booleanNode('enabled')
+                            ->defaultValue(function () {
+                                @trigger_error('Enabling the route generation feature is deprecated since FOSRestBundle 2.8.', E_USER_DEPRECATED);
```

Comment:
doesn't this trigger two deprecations now? one for the default case and one in normalization.

Reply:
I don't think so. IIRC normalization is only applied on user defined values and default values are only applied after the user config has been processed.